File: LTCB/MAX3SO.py
'''
Nhập vào 3 sô nguyên a,b,c. Tìm sô lớn nhât và sô nhỏ nhất trong 3 sô đó.

Ví dụ:
input
1 2 3

outpt:
3 1

'''
a,b,c = map(int, input().split())
# Cách 1: dùng Hàm
# max_num = max(a,b,c)
# min_num = min(a,b,c)
# print(max_num, min_num)

# Cách 2: Dùng Câu Lệnh Điều Kiện
if a >= b and a >= c:
    max_n = a
elif b >= a and b >= c:
    max_n = b
else:
    max_n = c

if a <= b and a <= c:
    min_n = a
elif b <= a and b <= c:
    min_n = b
else:
    min_n = c

print(max_n, min_n)

File: LTCB/C++/ReNhanh/Revise/MAX3SO.cpp
/*
Nhập vào 3 sô nguyên a,b,c. Tìm sô lớn nhât và sô nhỏ nhất trong 3 sô đó.

Ví dụ:
input
1 2 3

outpt:
3 1
*/

#include <iostream>
using namespace std;
int main(){
    int a, b, c;
    cin >> a >> b >> c;
    int max_n = a, min_n = a;
    if(b > max_n) max_n = b;
    if(c > max_n) max_n = c;
    if(b < min_n) min_n = b;
    if(c < min_n) min_n = c;
    cout << max_n << " " << min_n << endl;
    return 0;
}